auditlog/models/rule.py

```diff
@@ -267,6 +282,31 @@ def get_auditlog_fields(self, model):
             if (not f.compute and not f.related) or f.store
         )
 
+    def _make_export_data(self):
+        """Instanciate a create method that log its calls."""
+        self.ensure_one()
+        log_type = self.log_type
+        users_to_exclude = self.mapped("users_to_exclude_ids")
+
+        def export_data(self, fields_to_export):
+            res = export_data.origin(self, fields_to_export)
+            self = self.with_context(auditlog_disabled=True)
+            rule_model = self.env["auditlog.rule"]
+            if self.env.user in users_to_exclude:
+                return res
```

Comment:
Maybe can add a `sudo(demo_user)` export in the test as well, and beforehand exclude the demo user, then see if it refrains from logging.